zerver/lib/actions.py

```diff
@@ -1825,6 +1825,13 @@ def do_reactivate_user(user_profile):
 
     notify_created_user(user_profile)
 
+    if user_profile.is_bot:
+        event = dict(type="realm_bot", op="add",
+                     bot=dict(email=user_profile.email,
+                              user_id=user_profile.id,
+                              full_name=user_profile.full_name))
+        send_event(event, bot_owner_userids(user_profile))
+
```

Comment:
This should use `notify_created_bot` instead so the format is the same as a newly created bot.